libs/styles/src/lib/scss/components/_steps.scss

```diff
@@ -347,5 +356,52 @@
         @include cvi-screenreader-text;
       }
     }
+
+    #{$base}.is-scrollable {
+      & #{$base}__scroll-container {
+        @include cvi-breakpoint-down(sm) {
+          display: grid;
+        }
+
+        display: flex;
+        width: 100%;
+        overflow-x: auto;
+        scrollbar-width: none;
+        scroll-behavior: smooth;
+      }
+    }
+
+      & #{$base}__scroll-button {
```

Comment:
The extra `&` is not needed – it is automatically prepended.

Reply:
Removed extra `&`.